sprocs/check_assessment_access.sql

```diff
@@ -38,7 +38,9 @@ BEGIN
                     END)
             ELSE 'None'
         END AS credit_date_string,
-        aar.time_limit_min,
+        CASE WHEN aar.time_limit_min IS NULL THEN NULL
+             ELSE LEAST(aar.time_limit_min, EXTRACT(EPOCH FROM aar.end_date - now()) / 60)::integer
+        END AS time_limit_min,
```

Comment:
Indent style taken from [postgresql conditionals docs](https://www.postgresql.org/docs/7.4/functions-conditional.html) (though I didn't find any PL `CASE` blocks formatted this way)